apicurio-registry.yaml

```
@@ -71,12 +73,35 @@ subpackages:
       - working-directory: ui
         runs: |
           npm install
+          npm run clean
           npm audit fix --package-lock-only --legacy-peer-deps || true
           npm run build
+          npm run package
 
+          # Move UI assets to expected directory
           mkdir -p ${{targets.contextdir}}/opt/app-root/src
           cp -r ui-app/dist/* ${{targets.contextdir}}/opt/app-root/src/
 
+  - name: ${{package.name}}-nginx-config
+    description: nginx configuration files for upstream configuration
+    dependencies:
+      runtime:
+        - docker-nginx
+    pipeline:
+      - working-directory: ui
+        runs: |
+          # Configure certificate and key (per upstream)
+          mkdir -p /etc/pki/nginx/private
+          openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/pki/nginx/private/server.key -out /etc/pki/nginx/server.crt --batch
+          chown -R 1001:0 /etc/pki/nginx/ && chmod 755 /etc/pki/nginx/private/server.key /etc/pki/nginx/server.crt
+
+          # Copy configuration scripts (per upstream)
+          config_dest="${{targets.contextdir}}/usr/local/bin/"
+          install -Dm644 -o 1001 -g 0 .docker-scripts/create-config.cjs "${config_dest}/create-config.cjs"
+          install -Dm644 -o 1001 -g 0 .docker-scripts/update-base-href.cjs "${config_dest}/update-base-href.cjs"
+          install -Dm644 -o 1001 -g 0 .docker-scripts/entrypoint.sh "${config_dest}/entrypoint.sh"
+          install -Dm644 -o 1001 -g 0 .docker-scripts/nginx.conf "${config_dest}/etc/nginx/nginx.conf"
```

```suggestion
          config_dest="${{targets.contextdir}}/usr/local/bin/"
          install -Dm644 .docker-scripts/create-config.cjs "${config_dest}/create-config.cjs"
          install -Dm644 .docker-scripts/update-base-href.cjs "${config_dest}/update-base-href.cjs"
          install -Dm644 .docker-scripts/entrypoint.sh "${config_dest}/entrypoint.sh"
          install -Dm644 .docker-scripts/nginx.conf "${config_dest}/etc/nginx/nginx.conf"
```
I _feel_ like you can omit these, does it break anything?

---

Not at the moment, but I was doing this as per upstream's configuration